TF_Application/Assets/TouchFree/Tooling/Scripts/Cursors/DotCursor.cs

```diff
@@ -364,6 +362,8 @@ protected virtual void SetCursorLocalScale(float _scale)
         {
             cursorLocalScale = new Vector3(_scale, _scale, _scale);
             cursorBorder.transform.localScale = cursorLocalScale;
+
+            maxRingScale = cursorMaxRingSize;
```

Comment:
the use of maxRingScale can be removed and replaced with directly using cursorMaxRingScale